samples/matter/lock/README.rst

```diff
@@ -80,16 +80,20 @@ The configuration files are automatically attached to the build based on the bui
 To modify the configuration options, apply the modifications to the files with the appropriate suffix.
 See the table for information about available configuration types:
 
-+---------------------------+--------------------------------------+------------------------+------------------------------+
-| Config suffix             | Enabled feature                      | Enabling build option  | Supported boards             |
-+===========================+======================================+========================+==============================+
-| none                      | none (basic build)                   | none                   | ``nrf52840dk_nrf52840``      |
-|                           |                                      |                        | ``nrf5340dk_nrf5340_cpuapp`` |
-+---------------------------+--------------------------------------+------------------------+------------------------------+
-| :file:`_single_image_dfu` | Single-image Device Firmware Upgrade | ``-DBUILD_WITH_DFU=1`` | ``nrf52840dk_nrf52840``      |
-+---------------------------+--------------------------------------+------------------------+------------------------------+
-| :file:`_multi_image_dfu`  | Multi-image Device Firmware Upgrade  | ``-DBUILD_WITH_DFU=1`` | ``nrf5340dk_nrf5340_cpuapp`` |
-+---------------------------+--------------------------------------+------------------------+------------------------------+
++----------------------------------+--------------------------------------------------+-----------------------------+------------------------------+
+| Config suffix                    | Enabled feature                                  | Enabling build option       | Supported boards             |
++==================================+==================================================+=============================+==============================+
+| none                             | none (basic build)                               | none                        | ``nrf52840dk_nrf52840``      |
+|                                  |                                                  |                             | ``nrf5340dk_nrf5340_cpuapp`` |
++----------------------------------+--------------------------------------------------+-----------------------------+------------------------------+
+| :file:`_single_image_smp_dfu`    | Single-image Device Firmware Upgrade over SMP    | ``-DBUILD_WITH_DFU=BLE``    | ``nrf52840dk_nrf52840``      |
```

Comment:
Doesn't using this option enable DFU over Matter and over BLE at the same time?

Reply:
It does. It's mentioned in the section below. Should I add it here as well?

Reply:
Actually, added.